model/waterfall.go

```diff
@@ -0,0 +1,618 @@
+package model
+
+import (
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/evergreen/apimodels"
+	"github.com/evergreen-ci/evergreen/model/build"
+	"github.com/evergreen-ci/evergreen/model/task"
+	"github.com/evergreen-ci/evergreen/model/version"
+	"github.com/pkg/errors"
+)
+
+const (
+	WaterfallPerPageLimit  = 5
+	WaterfallBVFilterParam = "bv_filter"
+	WaterfallLimitParam    = "limit"
+	WaterfallSkipParam     = "skip"
```

Comment:
why are these in the model package? and why are they public?

Reply:
I have no idea where to put it, since this constants should be available for both UI service and REST V2. Including rest from ui service and vice versa may lead to circular dependencies and it also make no sense, because ui service and rest are on the same level (none is based or required each other to work)